tests/system/test_fim/test_fim_synchronization/test_fim_synchronization.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
```

Comment:
Add documentation

Reply:
Also, Add the changelog.md

Reply:
Done in commit [2c97b](https://github.com/wazuh/wazuh-qa/pull/2443/commits/2c97b0a71e51095651e5b519ede75b44571325a7) y [2413d](https://github.com/wazuh/wazuh-qa/pull/2443/commits/2413d0bf31fa4155181904016ba139024a71a835)